eslint.config.js

```diff
@@ -86,6 +86,7 @@ export default [
           ignoreRestSiblings: true,
         },
       ],
+      '@typescript-eslint/prefer-nullish-coalescing': 'off',
```

Comment:
Turned of the "new rule" because (from the [docs](https://typescript-eslint.io/rules/prefer-nullish-coalescing/#when-not-to-use-it)):

> This rule will not work as expected if [strictNullChecks](https://www.typescriptlang.org/tsconfig#strictNullChecks) is not enabled.